lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -318,6 +318,10 @@ public function create_item( $request ) {
 			return new WP_Error( 'rest_comment_exists', __( 'Cannot create existing comment.' ), array( 'status' => 400 ) );
 		}
 
+		if ( empty( $request['content'] ) || ! is_string( $request['content'] ) ) {
+			return new WP_Error( 'rest_comment_content_required', __( 'Missing comment content.' ), array( 'status' => 400 ) );
```

Comment:
Well in the schema `content` is an object, like title and content is in the Posts Controller....so how would we adapt the schema to handle the create/update Paramus that are not objects?